### ISSUE TITLE
`sudo ldconfig` on macOS??

### ISSUE THREAD

**Issue by hepcat72** [Pulse-Eight/platform#43]
The install procedure for macOS includes `sudo ldconfig`.  This doesn't appear to be a macOS thing.  What's the analog for mac users?

**Comment by aryairani** [comment]
btw you can `brew install llibcec` instead, if that helps

**Comment by opdenkamp** [comment]
You're right, and the short answer to your question is: **there isn't one — on OS X you just skip that step.** `ldconfig` refreshes the ELF linker cache on Linux and BSD; macOS uses `dyld`, which resolves install names at load time and has no cache to refresh. Nothing replaces it.

The README was simply wrong to print it under a heading that claims to cover OS X, and `apt-get` was in there too, which isn't a macOS thing either. Fixed in 81d99f2 — `apt-get` is now called out as Debian/Ubuntu, and `ldconfig` as Linux and BSD with an explicit note that OS X needs no equivalent.

While in there, a few other things in those same 39 lines turned out to be stale or broken, so they went in the same commit:

- `cmake 2.6 or better` → **3.12.0**, which is what `cmake_minimum_required` has actually required since d7faed1 (thanks @heitbaum, #53).
- `Visual Studio 2013 (v120) or 2015 (v140)` → **2022**, matching `windows/build.cmd`.
- `cd ` (no argument) → `cd windows`, where `build.cmd` actually lives (thanks @xXP4trickXx, #40).
- Every link was written `[text] (url)` — the space means **none of them ever rendered as links**.
- The instructions called `git clone` without listing `git` as a dependency, and `build-essential` doesn't include it. A clean Debian following the dependency list to the letter fell over on the very first command (`sh: git: not found`). It's `apt-get install git cmake build-essential` now.

The Linux recipe was then run end to end in a clean `debian:bookworm` container, installing exactly the dependencies the README lists and cloning the real repo: `cmake ..` → `make -j4` → `make install` → `ldconfig` all succeed, producing `/usr/local/lib/libp8-platform.a`.

(For anyone arriving from a search: `brew install libcec` is a fine way to *get libCEC*, but it doesn't answer how to build *this* library from source, which is what the README section is for.)
